io-backend/Dockerfile

```diff
@@ -10,15 +10,15 @@ ARG GITHUB_TOKEN
 ENV GITHUB_TOKEN=${GITHUB_TOKEN}
 
 RUN rm -rf * && \
-    git clone https://github.com/pagopa/io-backend.git /usr/src/app && \
-    git checkout $IO_BACKEND_COMMIT_SHA
+  git clone https://github.com/pagopa/io-backend.git /usr/src/app && \
+  git checkout $IO_BACKEND_COMMIT_SHA
 
 RUN ./scripts/init-npm.sh
 
 RUN yarn install && \
```

Comment:
should we add `--frozen-lockfile?`

Reply:
It could be a container more similar to production. I agree.

Reply:
The point is not `yarn.lock` file to change, the point is the container to run with **exactly same packages** at each installation

Reply:
Applied the change in fde259b .
I will open a PR for other containers so we can make them more stable.